iaso/tests/api/test_profiles.py

```diff
@@ -296,7 +296,7 @@ def test_profile_list_read_only_permissions(self):
         """GET /profiles/ with auth (user has read only permissions)"""
 
         self.client.force_authenticate(self.jane)
-        with self.assertNumQueries(17):
+        with self.assertNumQueries(11):
```

Comment:
I'm gonna write this comment here even though it's not related to this specific line: unfortunately I don't see any tests on user creation/update with `editable_org_unit_type_ids` :frowning_face: